components/post-meta/readme.md

```diff
@@ -0,0 +1,95 @@
+# Post Meta
+
+The `PostMeta` component is part of the suite of components that read/write data from the global post object or the current [`<PostContext />`](../post-context/). It has a composable declarative approach to allow you full control over the markup without needing to worry about the data handling at all.
+
+The Component allows you to showcase any piece of post-meta information of the current post.
+
+> **Warning**
+> In order for the Meta Value to show up it needs to be registered properly and have `show_in_rest` set to `true`.
+
+## Usage
+
+```js
+import { PostMeta } from '@10up/block-components';
+
+function BlockEdit() {
+
+    return (
+        <PostMeta metaKey="price" className="wp-block-example-block__price" />
+    )
+}
+```
+
+The component automatically tries to figure out the type of the meta value and dynamically displays either a text field, number control, or a toggle control.
+
+If you want to override this automatic type casting you can use the sub-components which expose the underlying type controls.
+
+> **Warning**
+> Currently only `string`, `number`, `boolean` for `single` meta values are supported.
+
+You can also completely customize the UI of the meta field by passing a render function to the `PostMeta` component:
```

Comment:
This is rather known as Function as child which is a pattern derivated from https://reactjs.org/docs/render-props.html I'd clarify this and add some links since this is not react entry level material. Feel free to adjust though!

```suggestion
You can also completely customize the UI of the meta field by passing a [render function as the children](https://reactpatterns.js.org/docs/function-as-child-component/) of the `PostMeta` component:
```